config/elastic-sink.properties

```diff
@@ -56,11 +57,6 @@ es.identifier.builder=com.ibm.eventstreams.connect.essink.builders.DefaultIdenti
 #es.max.failures=5
  
 # These converters control conversion of data between the internal Kafka Connect representation and the messages in Kafka.
```

Comment:
Do you want to edit this comment to say the connector expects json for the value converter? At the moment it is more open